2014/05/12 21:46:50 [error] [exception.CDbException] could not find driver
2014/05/12 21:46:50 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in D:\YiiRoot\db\CDbConnection.php:388
Stack trace:
#0 D:\YiiRoot\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\YiiRoot\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\YiiRoot\base\CModule.php(387): CDbConnection->init()
#3 D:\YiiRoot\base\CApplication.php(450): CModule->getComponent('db')
#4 D:\YiiRoot\db\ar\CActiveRecord.php(634): CApplication->getDb()
#5 D:\YiiRoot\db\ar\CActiveRecord.php(667): CActiveRecord->getDbConnection()
#6 D:\YiiRoot\db\ar\CActiveRecord.php(1595): CActiveRecord->getCommandBuilder()
#7 D:\YiiRoot\web\CActiveDataProvider.php(228): CActiveRecord->count(Object(CDbCriteria))
#8 D:\YiiRoot\web\CDataProvider.php(224): CActiveDataProvider->calculateTotalItemCount()
#9 D:\YiiRoot\web\CActiveDataProvider.php(178): CDataProvider->getTotalItemCount()
#10 D:\YiiRoot\web\CDataProvider.php(168): CActiveDataProvider->fetchData()
#11 D:\YiiRoot\zii\widgets\CBaseListView.php(111): CDataProvider->getData()
#12 D:\YiiRoot\zii\widgets\CListView.php(211): CBaseListView->init()
#13 D:\YiiRoot\web\CBaseController.php(147): CListView->init()
#14 D:\YiiRoot\web\CBaseController.php(172): CBaseController->createWidget('zii.widgets.CLi...', Array)
#15 D:\webroot\test.loc\protected\views\post\index.php(9): CBaseController->widget('zii.widgets.CLi...', Array)
#16 D:\YiiRoot\web\CBaseController.php(126): require('D:\\webroot\\test...')
#17 D:\YiiRoot\web\CBaseController.php(95): CBaseController->renderInternal('D:\\webroot\\test...', Array, true)
#18 D:\YiiRoot\web\CController.php(869): CBaseController->renderFile('D:\\webroot\\test...', Array, true)
#19 D:\YiiRoot\web\CController.php(782): CController->renderPartial('index', Array, true)
#20 D:\webroot\test.loc\protected\controllers\PostController.php(136): CController->render('index', Array)
#21 D:\YiiRoot\web\actions\CInlineAction.php(49): PostController->actionIndex()
#22 D:\YiiRoot\web\CController.php(308): CInlineAction->runWithParams(Array)
#23 D:\YiiRoot\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#24 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#25 D:\YiiRoot\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#26 D:\YiiRoot\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#27 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#28 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#29 D:\YiiRoot\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#30 D:\YiiRoot\web\CWebApplication.php(282): CController->run('')
#31 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('')
#32 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#33 D:\webroot\test.loc\index.php(11): CApplication->run()
#34 {main}
REQUEST_URI=/
---
2014/05/12 21:56:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in D:\YiiRoot\web\CWebApplication.php:286
Stack trace:
#0 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('gii')
#1 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\webroot\test.loc\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/gii
---
2014/05/12 21:56:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "r/gii".' in D:\YiiRoot\web\CWebApplication.php:286
Stack trace:
#0 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('r/gii')
#1 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\webroot\test.loc\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/r/gii
---
2014/05/12 21:57:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in D:\YiiRoot\web\CWebApplication.php:286
Stack trace:
#0 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('gii')
#1 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\webroot\test.loc\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/gii
---
2014/05/12 22:45:07 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'blog.category' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `category`.
2014/05/12 22:53:53 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'blog.category' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `category`.
2014/05/12 22:53:57 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'blog.category' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `category`.
2014/05/12 23:05:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "add".' in D:\YiiRoot\web\CController.php:483
Stack trace:
#0 D:\YiiRoot\web\CController.php(270): CController->missingAction('add')
#1 D:\YiiRoot\web\CWebApplication.php(282): CController->run('add')
#2 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/add')
#3 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\webroot\test.loc\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/category/add
---
2014/05/12 23:06:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "add".' in D:\YiiRoot\web\CController.php:483
Stack trace:
#0 D:\YiiRoot\web\CController.php(270): CController->missingAction('add')
#1 D:\YiiRoot\web\CWebApplication.php(282): CController->run('add')
#2 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/add/vi...')
#3 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\webroot\test.loc\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/category/add/view/1
---
2014/05/12 23:06:31 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in D:\YiiRoot\web\CController.php:336
Stack trace:
#0 D:\YiiRoot\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 D:\YiiRoot\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#3 D:\YiiRoot\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 D:\YiiRoot\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#7 D:\YiiRoot\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 D:\YiiRoot\web\CWebApplication.php(282): CController->run('view')
#9 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/view/1')
#10 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#11 D:\webroot\test.loc\index.php(11): CApplication->run()
#12 {main}
REQUEST_URI=/category/view/1
---
2014/05/12 23:06:33 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in D:\YiiRoot\web\CController.php:336
Stack trace:
#0 D:\YiiRoot\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 D:\YiiRoot\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#3 D:\YiiRoot\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 D:\YiiRoot\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#7 D:\YiiRoot\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 D:\YiiRoot\web\CWebApplication.php(282): CController->run('view')
#9 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/view/1')
#10 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#11 D:\webroot\test.loc\index.php(11): CApplication->run()
#12 {main}
REQUEST_URI=/category/view/1
---
2014/05/12 23:06:39 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in D:\YiiRoot\web\CController.php:336
Stack trace:
#0 D:\YiiRoot\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 D:\YiiRoot\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#3 D:\YiiRoot\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 D:\YiiRoot\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#7 D:\YiiRoot\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 D:\YiiRoot\web\CWebApplication.php(282): CController->run('view')
#9 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/view')
#10 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#11 D:\webroot\test.loc\index.php(11): CApplication->run()
#12 {main}
REQUEST_URI=/category/view
---
2014/05/12 23:11:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "vew".' in D:\YiiRoot\web\CController.php:483
Stack trace:
#0 D:\YiiRoot\web\CController.php(270): CController->missingAction('vew')
#1 D:\YiiRoot\web\CWebApplication.php(282): CController->run('vew')
#2 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/vew/1')
#3 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\webroot\test.loc\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/category/vew/1
---
2014/05/12 23:12:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "2".' in D:\YiiRoot\web\CController.php:483
Stack trace:
#0 D:\YiiRoot\web\CController.php(270): CController->missingAction('2')
#1 D:\YiiRoot\web\CWebApplication.php(282): CController->run('2')
#2 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/2/vew')
#3 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\webroot\test.loc\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/category/2/vew
---
2014/05/12 23:12:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "1".' in D:\YiiRoot\web\CController.php:483
Stack trace:
#0 D:\YiiRoot\web\CController.php(270): CController->missingAction('1')
#1 D:\YiiRoot\web\CWebApplication.php(282): CController->run('1')
#2 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/1/vew')
#3 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\webroot\test.loc\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/category/1/vew
---
2014/05/12 23:12:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "1".' in D:\YiiRoot\web\CController.php:483
Stack trace:
#0 D:\YiiRoot\web\CController.php(270): CController->missingAction('1')
#1 D:\YiiRoot\web\CWebApplication.php(282): CController->run('1')
#2 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/1/vew')
#3 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\webroot\test.loc\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/category/1/vew
---
2014/05/12 23:15:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "1".' in D:\YiiRoot\web\CController.php:483
Stack trace:
#0 D:\YiiRoot\web\CController.php(270): CController->missingAction('1')
#1 D:\YiiRoot\web\CWebApplication.php(282): CController->run('1')
#2 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/1/vew')
#3 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\webroot\test.loc\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/category/1/vew
---
2014/05/12 23:15:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "2".' in D:\YiiRoot\web\CController.php:483
Stack trace:
#0 D:\YiiRoot\web\CController.php(270): CController->missingAction('2')
#1 D:\YiiRoot\web\CWebApplication.php(282): CController->run('2')
#2 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/2/vew')
#3 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\webroot\test.loc\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/category/2/vew
---
2014/05/12 23:15:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "vew".' in D:\YiiRoot\web\CController.php:483
Stack trace:
#0 D:\YiiRoot\web\CController.php(270): CController->missingAction('vew')
#1 D:\YiiRoot\web\CWebApplication.php(282): CController->run('vew')
#2 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/vew/1')
#3 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\webroot\test.loc\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/category/vew/1
---
2014/05/12 23:15:22 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in D:\YiiRoot\web\CController.php:336
Stack trace:
#0 D:\YiiRoot\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 D:\YiiRoot\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#3 D:\YiiRoot\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 D:\YiiRoot\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#7 D:\YiiRoot\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 D:\YiiRoot\web\CWebApplication.php(282): CController->run('view')
#9 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/view/1')
#10 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#11 D:\webroot\test.loc\index.php(11): CApplication->run()
#12 {main}
REQUEST_URI=/category/view/1
---
2014/05/12 23:15:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "1".' in D:\YiiRoot\web\CController.php:483
Stack trace:
#0 D:\YiiRoot\web\CController.php(270): CController->missingAction('1')
#1 D:\YiiRoot\web\CWebApplication.php(282): CController->run('1')
#2 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/1/view')
#3 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\webroot\test.loc\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/category/1/view
---
2014/05/12 23:15:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "2".' in D:\YiiRoot\web\CController.php:483
Stack trace:
#0 D:\YiiRoot\web\CController.php(270): CController->missingAction('2')
#1 D:\YiiRoot\web\CWebApplication.php(282): CController->run('2')
#2 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/2/view')
#3 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\webroot\test.loc\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/category/2/view
---
2014/05/12 23:15:38 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in D:\YiiRoot\web\CController.php:336
Stack trace:
#0 D:\YiiRoot\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 D:\YiiRoot\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#3 D:\YiiRoot\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 D:\YiiRoot\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#7 D:\YiiRoot\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 D:\YiiRoot\web\CWebApplication.php(282): CController->run('view')
#9 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/view/2')
#10 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#11 D:\webroot\test.loc\index.php(11): CApplication->run()
#12 {main}
REQUEST_URI=/category/view/2
---
2014/05/12 23:15:41 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in D:\YiiRoot\web\CController.php:336
Stack trace:
#0 D:\YiiRoot\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 D:\YiiRoot\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#3 D:\YiiRoot\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 D:\YiiRoot\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#7 D:\YiiRoot\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 D:\YiiRoot\web\CWebApplication.php(282): CController->run('view')
#9 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/view/2')
#10 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#11 D:\webroot\test.loc\index.php(11): CApplication->run()
#12 {main}
REQUEST_URI=/category/view/2
---
2014/05/12 23:17:26 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in D:\YiiRoot\web\CController.php:336
Stack trace:
#0 D:\YiiRoot\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 D:\YiiRoot\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#3 D:\YiiRoot\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 D:\YiiRoot\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#7 D:\YiiRoot\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 D:\YiiRoot\web\CWebApplication.php(282): CController->run('view')
#9 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/view/2')
#10 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#11 D:\webroot\test.loc\index.php(11): CApplication->run()
#12 {main}
REQUEST_URI=/category/view/2
---
2014/05/12 23:17:29 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in D:\YiiRoot\web\CController.php:336
Stack trace:
#0 D:\YiiRoot\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 D:\YiiRoot\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#3 D:\YiiRoot\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 D:\YiiRoot\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#7 D:\YiiRoot\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 D:\YiiRoot\web\CWebApplication.php(282): CController->run('view')
#9 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/view/1')
#10 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#11 D:\webroot\test.loc\index.php(11): CApplication->run()
#12 {main}
REQUEST_URI=/category/view/1
---
2014/05/12 23:17:35 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in D:\YiiRoot\web\CController.php:336
Stack trace:
#0 D:\YiiRoot\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 D:\YiiRoot\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#3 D:\YiiRoot\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 D:\YiiRoot\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#7 D:\YiiRoot\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 D:\YiiRoot\web\CWebApplication.php(282): CController->run('view')
#9 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/view')
#10 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#11 D:\webroot\test.loc\index.php(11): CApplication->run()
#12 {main}
REQUEST_URI=/category/view
---
2014/05/12 23:18:16 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xD0\x9F\xD0\xB5\xD1\x80...' for column 'name' at row 1. The SQL statement executed was: INSERT INTO `tbl_category` (`name`) VALUES (:yp0).
2014/05/12 23:18:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xD0\x9F\xD0\xB5\xD1\x80...' for column 'name' at row 1' in D:\YiiRoot\db\CDbCommand.php:358
Stack trace:
#0 D:\YiiRoot\db\ar\CActiveRecord.php(1077): CDbCommand->execute()
#1 D:\YiiRoot\db\ar\CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 D:\webroot\test.loc\protected\controllers\CategoryController.php(73): CActiveRecord->save()
#3 D:\YiiRoot\web\actions\CInlineAction.php(49): CategoryController->actionCreate()
#4 D:\YiiRoot\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 D:\YiiRoot\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#7 D:\YiiRoot\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 D:\YiiRoot\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#11 D:\YiiRoot\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\YiiRoot\web\CWebApplication.php(282): CController->run('create')
#13 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/create')
#14 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#15 D:\webroot\test.loc\index.php(11): CApplication->run()
#16 {main}
REQUEST_URI=/category/create
HTTP_REFERER=http://test.loc/category/create
---
2014/05/12 23:21:32 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xD0\x9F\xD0\xB5\xD1\x80...' for column 'name' at row 1. The SQL statement executed was: INSERT INTO `tbl_category` (`name`) VALUES (:yp0).
2014/05/12 23:21:32 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xD0\x9F\xD0\xB5\xD1\x80...' for column 'name' at row 1' in D:\YiiRoot\db\CDbCommand.php:358
Stack trace:
#0 D:\YiiRoot\db\ar\CActiveRecord.php(1077): CDbCommand->execute()
#1 D:\YiiRoot\db\ar\CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 D:\webroot\test.loc\protected\controllers\CategoryController.php(73): CActiveRecord->save()
#3 D:\YiiRoot\web\actions\CInlineAction.php(49): CategoryController->actionCreate()
#4 D:\YiiRoot\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 D:\YiiRoot\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#7 D:\YiiRoot\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 D:\YiiRoot\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#11 D:\YiiRoot\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\YiiRoot\web\CWebApplication.php(282): CController->run('create')
#13 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/create')
#14 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#15 D:\webroot\test.loc\index.php(11): CApplication->run()
#16 {main}
REQUEST_URI=/category/create
HTTP_REFERER=http://test.loc/category/create
---
2014/05/12 23:22:49 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xD0\x9F\xD0\xB5\xD1\x80...' for column 'name' at row 1. The SQL statement executed was: INSERT INTO `tbl_category` (`name`) VALUES (:yp0).
2014/05/12 23:22:49 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xD0\x9F\xD0\xB5\xD1\x80...' for column 'name' at row 1' in D:\YiiRoot\db\CDbCommand.php:358
Stack trace:
#0 D:\YiiRoot\db\ar\CActiveRecord.php(1077): CDbCommand->execute()
#1 D:\YiiRoot\db\ar\CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 D:\webroot\test.loc\protected\controllers\CategoryController.php(73): CActiveRecord->save()
#3 D:\YiiRoot\web\actions\CInlineAction.php(49): CategoryController->actionCreate()
#4 D:\YiiRoot\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 D:\YiiRoot\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#7 D:\YiiRoot\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 D:\YiiRoot\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#11 D:\YiiRoot\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\YiiRoot\web\CWebApplication.php(282): CController->run('create')
#13 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/create')
#14 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#15 D:\webroot\test.loc\index.php(11): CApplication->run()
#16 {main}
REQUEST_URI=/category/create
HTTP_REFERER=http://test.loc/category/create
---
2014/05/12 23:25:35 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xD0\x9F\xD0\xB5\xD1\x80...' for column 'name' at row 1. The SQL statement executed was: INSERT INTO `tbl_category` (`name`) VALUES (:yp0).
2014/05/12 23:25:35 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xD0\x9F\xD0\xB5\xD1\x80...' for column 'name' at row 1' in D:\YiiRoot\db\CDbCommand.php:358
Stack trace:
#0 D:\YiiRoot\db\ar\CActiveRecord.php(1077): CDbCommand->execute()
#1 D:\YiiRoot\db\ar\CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 D:\webroot\test.loc\protected\controllers\CategoryController.php(73): CActiveRecord->save()
#3 D:\YiiRoot\web\actions\CInlineAction.php(49): CategoryController->actionCreate()
#4 D:\YiiRoot\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 D:\YiiRoot\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#7 D:\YiiRoot\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 D:\YiiRoot\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#11 D:\YiiRoot\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\YiiRoot\web\CWebApplication.php(282): CController->run('create')
#13 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/create')
#14 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#15 D:\webroot\test.loc\index.php(11): CApplication->run()
#16 {main}
REQUEST_URI=/category/create
HTTP_REFERER=http://test.loc/category/create
---
2014/05/12 23:27:04 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xD0\x9F\xD0\xB5\xD1\x80...' for column 'name' at row 1. The SQL statement executed was: INSERT INTO `tbl_category` (`name`) VALUES (:yp0).
2014/05/12 23:27:04 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xD0\x9F\xD0\xB5\xD1\x80...' for column 'name' at row 1' in D:\YiiRoot\db\CDbCommand.php:358
Stack trace:
#0 D:\YiiRoot\db\ar\CActiveRecord.php(1077): CDbCommand->execute()
#1 D:\YiiRoot\db\ar\CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 D:\webroot\test.loc\protected\controllers\CategoryController.php(73): CActiveRecord->save()
#3 D:\YiiRoot\web\actions\CInlineAction.php(49): CategoryController->actionCreate()
#4 D:\YiiRoot\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 D:\YiiRoot\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#7 D:\YiiRoot\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 D:\YiiRoot\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#11 D:\YiiRoot\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\YiiRoot\web\CWebApplication.php(282): CController->run('create')
#13 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/create')
#14 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#15 D:\webroot\test.loc\index.php(11): CApplication->run()
#16 {main}
REQUEST_URI=/category/create
HTTP_REFERER=http://test.loc/category/create
---
2014/05/12 23:27:07 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xD0\x9F\xD0\xB5\xD1\x80...' for column 'name' at row 1. The SQL statement executed was: INSERT INTO `tbl_category` (`name`) VALUES (:yp0).
2014/05/12 23:27:07 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xD0\x9F\xD0\xB5\xD1\x80...' for column 'name' at row 1' in D:\YiiRoot\db\CDbCommand.php:358
Stack trace:
#0 D:\YiiRoot\db\ar\CActiveRecord.php(1077): CDbCommand->execute()
#1 D:\YiiRoot\db\ar\CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 D:\webroot\test.loc\protected\controllers\CategoryController.php(73): CActiveRecord->save()
#3 D:\YiiRoot\web\actions\CInlineAction.php(49): CategoryController->actionCreate()
#4 D:\YiiRoot\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 D:\YiiRoot\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#7 D:\YiiRoot\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 D:\YiiRoot\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#11 D:\YiiRoot\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\YiiRoot\web\CWebApplication.php(282): CController->run('create')
#13 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/create')
#14 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#15 D:\webroot\test.loc\index.php(11): CApplication->run()
#16 {main}
REQUEST_URI=/category/create
HTTP_REFERER=http://test.loc/category/create
---
2014/05/13 09:27:06 [error] [exception.CDbException] SQLSTATE[HY000] [1049] Unknown database 'blog'
2014/05/13 09:27:06 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection.' in D:\YiiRoot\db\CDbConnection.php:388
Stack trace:
#0 D:\YiiRoot\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\YiiRoot\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\YiiRoot\base\CModule.php(387): CDbConnection->init()
#3 D:\YiiRoot\base\CApplication.php(450): CModule->getComponent('db')
#4 D:\YiiRoot\db\ar\CActiveRecord.php(634): CApplication->getDb()
#5 D:\YiiRoot\db\ar\CActiveRecord.php(667): CActiveRecord->getDbConnection()
#6 D:\YiiRoot\db\ar\CActiveRecord.php(1595): CActiveRecord->getCommandBuilder()
#7 D:\YiiRoot\web\CActiveDataProvider.php(228): CActiveRecord->count(Object(CDbCriteria))
#8 D:\YiiRoot\web\CDataProvider.php(224): CActiveDataProvider->calculateTotalItemCount()
#9 D:\YiiRoot\web\CActiveDataProvider.php(178): CDataProvider->getTotalItemCount()
#10 D:\YiiRoot\web\CDataProvider.php(168): CActiveDataProvider->fetchData()
#11 D:\YiiRoot\zii\widgets\CBaseListView.php(111): CDataProvider->getData()
#12 D:\YiiRoot\zii\widgets\CListView.php(211): CBaseListView->init()
#13 D:\YiiRoot\web\CBaseController.php(147): CListView->init()
#14 D:\YiiRoot\web\CBaseController.php(172): CBaseController->createWidget('zii.widgets.CLi...', Array)
#15 D:\Webroot\test.loc\protected\views\post\index.php(9): CBaseController->widget('zii.widgets.CLi...', Array)
#16 D:\YiiRoot\web\CBaseController.php(126): require('D:\\Webroot\\test...')
#17 D:\YiiRoot\web\CBaseController.php(95): CBaseController->renderInternal('D:\\Webroot\\test...', Array, true)
#18 D:\YiiRoot\web\CController.php(869): CBaseController->renderFile('D:\\Webroot\\test...', Array, true)
#19 D:\YiiRoot\web\CController.php(782): CController->renderPartial('index', Array, true)
#20 D:\Webroot\test.loc\protected\controllers\PostController.php(136): CController->render('index', Array)
#21 D:\YiiRoot\web\actions\CInlineAction.php(49): PostController->actionIndex()
#22 D:\YiiRoot\web\CController.php(308): CInlineAction->runWithParams(Array)
#23 D:\YiiRoot\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#24 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#25 D:\YiiRoot\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#26 D:\YiiRoot\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#27 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#28 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#29 D:\YiiRoot\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#30 D:\YiiRoot\web\CWebApplication.php(282): CController->run('')
#31 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('')
#32 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#33 D:\Webroot\test.loc\index.php(11): CApplication->run()
#34 {main}
REQUEST_URI=/
---
2014/05/13 09:27:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\YiiRoot\web\CWebApplication.php:286
Stack trace:
#0 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\Webroot\test.loc\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2014/05/13 09:27:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in D:\YiiRoot\web\CWebApplication.php:286
Stack trace:
#0 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\Webroot\test.loc\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2014/05/13 09:29:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "categoty".' in D:\YiiRoot\web\CWebApplication.php:286
Stack trace:
#0 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('categoty')
#1 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\Webroot\test.loc\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/categoty
---
2014/05/13 09:29:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "reate".' in D:\YiiRoot\web\CController.php:483
Stack trace:
#0 D:\YiiRoot\web\CController.php(270): CController->missingAction('reate')
#1 D:\YiiRoot\web\CWebApplication.php(282): CController->run('reate')
#2 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/reate')
#3 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\Webroot\test.loc\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/category/reate
---
2014/05/13 09:48:13 [error] [php] include(category_id.php): failed to open stream: No such file or directory (D:\YiiRoot\YiiBase.php:427)
Stack trace:
#0 D:\YiiRoot\validators\CValidator.php(186): import()
#1 D:\YiiRoot\base\CModel.php(287): createValidator()
#2 D:\YiiRoot\base\CModel.php(260): Post->createValidators()
#3 D:\YiiRoot\base\CModel.php(304): Post->getValidators()
#4 D:\YiiRoot\web\helpers\CHtml.php(1414): Post->isAttributeRequired()
#5 D:\YiiRoot\web\widgets\CActiveForm.php(597): activeLabelEx()
#6 D:\Webroot\test.loc\protected\views\post\_form.php(10): CActiveForm->labelEx()
#7 D:\YiiRoot\web\CBaseController.php(126): require()
#8 D:\YiiRoot\web\CBaseController.php(95): PostController->renderInternal()
#9 D:\YiiRoot\web\CController.php(869): PostController->renderFile()
#10 D:\Webroot\test.loc\protected\views\post\create.php(8): PostController->renderPartial()
#11 D:\YiiRoot\web\CBaseController.php(126): require()
#12 D:\YiiRoot\web\CBaseController.php(95): PostController->renderInternal()
#13 D:\YiiRoot\web\CController.php(869): PostController->renderFile()
#14 D:\YiiRoot\web\CController.php(782): PostController->renderPartial()
#15 D:\Webroot\test.loc\protected\controllers\PostController.php(73): PostController->render()
#16 D:\YiiRoot\web\actions\CInlineAction.php(49): PostController->actionCreate()
#17 D:\YiiRoot\web\CController.php(308): CInlineAction->runWithParams()
#18 D:\YiiRoot\web\filters\CFilterChain.php(133): PostController->runAction()
#19 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#20 D:\YiiRoot\web\CController.php(1145): CAccessControlFilter->filter()
#21 D:\YiiRoot\web\filters\CInlineFilter.php(58): PostController->filterAccessControl()
#22 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#23 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#24 D:\YiiRoot\web\CController.php(265): PostController->runActionWithFilters()
#25 D:\YiiRoot\web\CWebApplication.php(282): PostController->run()
#26 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController()
#27 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#28 D:\Webroot\test.loc\index.php(11): CWebApplication->run()
REQUEST_URI=/post/create
2014/05/13 10:03:44 [error] [php] Trying to get property of non-object (D:\Webroot\test.loc\protected\views\post\_view.php:9)
Stack trace:
#0 D:\YiiRoot\zii\widgets\CListView.php(291): PostController->renderFile()
#1 D:\YiiRoot\zii\widgets\CBaseListView.php(167): CListView->renderItems()
#2 unknown(0): CListView->renderSection()
#3 D:\YiiRoot\zii\widgets\CBaseListView.php(150): preg_replace_callback()
#4 D:\YiiRoot\zii\widgets\CBaseListView.php(135): CListView->renderContent()
#5 D:\YiiRoot\web\CBaseController.php(173): CListView->run()
#6 D:\Webroot\test.loc\protected\views\post\index.php(9): PostController->widget()
#7 D:\YiiRoot\web\CBaseController.php(126): require()
#8 D:\YiiRoot\web\CBaseController.php(95): PostController->renderInternal()
#9 D:\YiiRoot\web\CController.php(869): PostController->renderFile()
#10 D:\YiiRoot\web\CController.php(782): PostController->renderPartial()
#11 D:\Webroot\test.loc\protected\controllers\PostController.php(136): PostController->render()
#12 D:\YiiRoot\web\actions\CInlineAction.php(49): PostController->actionIndex()
#13 D:\YiiRoot\web\CController.php(308): CInlineAction->runWithParams()
#14 D:\YiiRoot\web\filters\CFilterChain.php(133): PostController->runAction()
#15 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#16 D:\YiiRoot\web\CController.php(1145): CAccessControlFilter->filter()
#17 D:\YiiRoot\web\filters\CInlineFilter.php(58): PostController->filterAccessControl()
#18 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#19 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#20 D:\YiiRoot\web\CController.php(265): PostController->runActionWithFilters()
#21 D:\YiiRoot\web\CWebApplication.php(282): PostController->run()
#22 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController()
#23 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#24 D:\Webroot\test.loc\index.php(11): CWebApplication->run()
REQUEST_URI=/post/
2014/05/13 10:24:19 [error] [php] htmlspecialchars() expects parameter 1 to be string, array given (D:\YiiRoot\web\helpers\CHtml.php:103)
Stack trace:
#0 D:\YiiRoot\web\helpers\CHtml.php(158): renderAttributes()
#1 D:\YiiRoot\web\helpers\CHtml.php(1822): tag()
#2 D:\YiiRoot\web\widgets\CActiveForm.php(847): activeDropDownList()
#3 D:\Webroot\test.loc\protected\views\post\_form.php(54): CActiveForm->dropDownList()
#4 D:\YiiRoot\web\CBaseController.php(126): require()
#5 D:\YiiRoot\web\CBaseController.php(95): PostController->renderInternal()
#6 D:\YiiRoot\web\CController.php(869): PostController->renderFile()
#7 D:\Webroot\test.loc\protected\views\post\update.php(10): PostController->renderPartial()
#8 D:\YiiRoot\web\CBaseController.php(126): require()
#9 D:\YiiRoot\web\CBaseController.php(95): PostController->renderInternal()
#10 D:\YiiRoot\web\CController.php(869): PostController->renderFile()
#11 D:\YiiRoot\web\CController.php(782): PostController->renderPartial()
#12 D:\Webroot\test.loc\protected\controllers\PostController.php(92): PostController->render()
#13 D:\YiiRoot\web\actions\CInlineAction.php(49): PostController->actionUpdate()
#14 D:\YiiRoot\web\CController.php(308): CInlineAction->runWithParams()
#15 D:\YiiRoot\web\filters\CFilterChain.php(133): PostController->runAction()
#16 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#17 D:\YiiRoot\web\CController.php(1145): CAccessControlFilter->filter()
#18 D:\YiiRoot\web\filters\CInlineFilter.php(58): PostController->filterAccessControl()
#19 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#20 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#21 D:\YiiRoot\web\CController.php(265): PostController->runActionWithFilters()
#22 D:\YiiRoot\web\CWebApplication.php(282): PostController->run()
#23 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController()
#24 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#25 D:\Webroot\test.loc\index.php(11): CWebApplication->run()
REQUEST_URI=/post/update?id=5
2014/05/13 10:25:00 [error] [php] htmlspecialchars() expects parameter 1 to be string, array given (D:\YiiRoot\web\helpers\CHtml.php:103)
Stack trace:
#0 D:\YiiRoot\web\helpers\CHtml.php(158): renderAttributes()
#1 D:\YiiRoot\web\helpers\CHtml.php(1822): tag()
#2 D:\YiiRoot\web\widgets\CActiveForm.php(847): activeDropDownList()
#3 D:\Webroot\test.loc\protected\views\post\_form.php(54): CActiveForm->dropDownList()
#4 D:\YiiRoot\web\CBaseController.php(126): require()
#5 D:\YiiRoot\web\CBaseController.php(95): PostController->renderInternal()
#6 D:\YiiRoot\web\CController.php(869): PostController->renderFile()
#7 D:\Webroot\test.loc\protected\views\post\update.php(10): PostController->renderPartial()
#8 D:\YiiRoot\web\CBaseController.php(126): require()
#9 D:\YiiRoot\web\CBaseController.php(95): PostController->renderInternal()
#10 D:\YiiRoot\web\CController.php(869): PostController->renderFile()
#11 D:\YiiRoot\web\CController.php(782): PostController->renderPartial()
#12 D:\Webroot\test.loc\protected\controllers\PostController.php(92): PostController->render()
#13 D:\YiiRoot\web\actions\CInlineAction.php(49): PostController->actionUpdate()
#14 D:\YiiRoot\web\CController.php(308): CInlineAction->runWithParams()
#15 D:\YiiRoot\web\filters\CFilterChain.php(133): PostController->runAction()
#16 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#17 D:\YiiRoot\web\CController.php(1145): CAccessControlFilter->filter()
#18 D:\YiiRoot\web\filters\CInlineFilter.php(58): PostController->filterAccessControl()
#19 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#20 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#21 D:\YiiRoot\web\CController.php(265): PostController->runActionWithFilters()
#22 D:\YiiRoot\web\CWebApplication.php(282): PostController->run()
#23 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController()
#24 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#25 D:\Webroot\test.loc\index.php(11): CWebApplication->run()
REQUEST_URI=/post/update?id=5
2014/05/13 10:28:51 [error] [php] htmlspecialchars() expects parameter 1 to be string, array given (D:\YiiRoot\web\helpers\CHtml.php:103)
Stack trace:
#0 D:\YiiRoot\web\helpers\CHtml.php(158): renderAttributes()
#1 D:\YiiRoot\web\helpers\CHtml.php(1822): tag()
#2 D:\YiiRoot\web\widgets\CActiveForm.php(847): activeDropDownList()
#3 D:\Webroot\test.loc\protected\views\post\_form.php(54): CActiveForm->dropDownList()
#4 D:\YiiRoot\web\CBaseController.php(126): require()
#5 D:\YiiRoot\web\CBaseController.php(95): PostController->renderInternal()
#6 D:\YiiRoot\web\CController.php(869): PostController->renderFile()
#7 D:\Webroot\test.loc\protected\views\post\update.php(10): PostController->renderPartial()
#8 D:\YiiRoot\web\CBaseController.php(126): require()
#9 D:\YiiRoot\web\CBaseController.php(95): PostController->renderInternal()
#10 D:\YiiRoot\web\CController.php(869): PostController->renderFile()
#11 D:\YiiRoot\web\CController.php(782): PostController->renderPartial()
#12 D:\Webroot\test.loc\protected\controllers\PostController.php(92): PostController->render()
#13 D:\YiiRoot\web\actions\CInlineAction.php(49): PostController->actionUpdate()
#14 D:\YiiRoot\web\CController.php(308): CInlineAction->runWithParams()
#15 D:\YiiRoot\web\filters\CFilterChain.php(133): PostController->runAction()
#16 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#17 D:\YiiRoot\web\CController.php(1145): CAccessControlFilter->filter()
#18 D:\YiiRoot\web\filters\CInlineFilter.php(58): PostController->filterAccessControl()
#19 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#20 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#21 D:\YiiRoot\web\CController.php(265): PostController->runActionWithFilters()
#22 D:\YiiRoot\web\CWebApplication.php(282): PostController->run()
#23 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController()
#24 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#25 D:\Webroot\test.loc\index.php(11): CWebApplication->run()
REQUEST_URI=/post/update?id=5
2014/05/13 11:11:39 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in D:\YiiRoot\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 D:\YiiRoot\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 D:\YiiRoot\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 D:\YiiRoot\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 D:\YiiRoot\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#6 D:\YiiRoot\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 D:\YiiRoot\web\CWebApplication.php(282): CController->run('admin')
#8 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/admin')
#9 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#10 D:\Webroot\test.loc\index.php(11): CApplication->run()
#11 {main}
REQUEST_URI=/category/admin
HTTP_REFERER=http://test.loc/category/create
---
2014/05/13 11:12:01 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in D:\YiiRoot\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 D:\YiiRoot\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 D:\YiiRoot\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 D:\YiiRoot\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 D:\YiiRoot\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#6 D:\YiiRoot\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 D:\YiiRoot\web\CWebApplication.php(282): CController->run('admin')
#8 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController('category/admin')
#9 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#10 D:\Webroot\test.loc\index.php(11): CApplication->run()
#11 {main}
REQUEST_URI=/category/admin
HTTP_REFERER=http://test.loc/category/create
---
2014/05/13 11:21:01 [error] [php] Undefined variable: data (D:\Webroot\test.loc\protected\views\post\admin.php:24)
Stack trace:
#0 D:\YiiRoot\web\CController.php(869): PostController->renderFile()
#1 D:\YiiRoot\web\CController.php(782): PostController->renderPartial()
#2 D:\Webroot\test.loc\protected\controllers\PostController.php(149): PostController->render()
#3 D:\YiiRoot\web\actions\CInlineAction.php(49): PostController->actionAdmin()
#4 D:\YiiRoot\web\CController.php(308): CInlineAction->runWithParams()
#5 D:\YiiRoot\web\filters\CFilterChain.php(133): PostController->runAction()
#6 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#7 D:\YiiRoot\web\CController.php(1145): CAccessControlFilter->filter()
#8 D:\YiiRoot\web\filters\CInlineFilter.php(58): PostController->filterAccessControl()
#9 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#11 D:\YiiRoot\web\CController.php(265): PostController->runActionWithFilters()
#12 D:\YiiRoot\web\CWebApplication.php(282): PostController->run()
#13 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController()
#14 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#15 D:\Webroot\test.loc\index.php(11): CWebApplication->run()
REQUEST_URI=/post/admin
2014/05/13 11:21:38 [error] [php] Trying to get property of non-object (D:\YiiRoot\base\CComponent.php(612) : eval()'d code:1)
Stack trace:
#0 D:\YiiRoot\zii\widgets\grid\CGridColumn.php(151): CDataColumn->renderDataCellContent()
#1 D:\YiiRoot\zii\widgets\grid\CGridView.php(616): CDataColumn->renderDataCell()
#2 D:\YiiRoot\zii\widgets\grid\CGridView.php(572): CGridView->renderTableRow()
#3 D:\YiiRoot\zii\widgets\grid\CGridView.php(482): CGridView->renderTableBody()
#4 D:\YiiRoot\zii\widgets\CBaseListView.php(167): CGridView->renderItems()
#5 unknown(0): CGridView->renderSection()
#6 D:\YiiRoot\zii\widgets\CBaseListView.php(150): preg_replace_callback()
#7 D:\YiiRoot\zii\widgets\CBaseListView.php(135): CGridView->renderContent()
#8 D:\YiiRoot\web\CBaseController.php(173): CGridView->run()
#9 D:\Webroot\test.loc\protected\views\post\admin.php(36): PostController->widget()
#10 D:\YiiRoot\web\CBaseController.php(126): require()
#11 D:\YiiRoot\web\CBaseController.php(95): PostController->renderInternal()
#12 D:\YiiRoot\web\CController.php(869): PostController->renderFile()
#13 D:\YiiRoot\web\CController.php(782): PostController->renderPartial()
#14 D:\Webroot\test.loc\protected\controllers\PostController.php(149): PostController->render()
#15 D:\YiiRoot\web\actions\CInlineAction.php(49): PostController->actionAdmin()
#16 D:\YiiRoot\web\CController.php(308): CInlineAction->runWithParams()
#17 D:\YiiRoot\web\filters\CFilterChain.php(133): PostController->runAction()
#18 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#19 D:\YiiRoot\web\CController.php(1145): CAccessControlFilter->filter()
#20 D:\YiiRoot\web\filters\CInlineFilter.php(58): PostController->filterAccessControl()
#21 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#22 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#23 D:\YiiRoot\web\CController.php(265): PostController->runActionWithFilters()
#24 D:\YiiRoot\web\CWebApplication.php(282): PostController->run()
#25 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController()
#26 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#27 D:\Webroot\test.loc\index.php(11): CWebApplication->run()
REQUEST_URI=/post/admin
2014/05/13 11:26:00 [error] [php] Missing argument 3 for CHtml::listData(), called in D:\Webroot\test.loc\protected\views\post\admin.php on line 25 and defined (D:\YiiRoot\web\helpers\CHtml.php:2073)
Stack trace:
#0 D:\YiiRoot\web\CBaseController.php(95): PostController->renderInternal()
#1 D:\YiiRoot\web\CController.php(869): PostController->renderFile()
#2 D:\YiiRoot\web\CController.php(782): PostController->renderPartial()
#3 D:\Webroot\test.loc\protected\controllers\PostController.php(149): PostController->render()
#4 D:\YiiRoot\web\actions\CInlineAction.php(49): PostController->actionAdmin()
#5 D:\YiiRoot\web\CController.php(308): CInlineAction->runWithParams()
#6 D:\YiiRoot\web\filters\CFilterChain.php(133): PostController->runAction()
#7 D:\YiiRoot\web\filters\CFilter.php(40): CFilterChain->run()
#8 D:\YiiRoot\web\CController.php(1145): CAccessControlFilter->filter()
#9 D:\YiiRoot\web\filters\CInlineFilter.php(58): PostController->filterAccessControl()
#10 D:\YiiRoot\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#11 D:\YiiRoot\web\CController.php(291): CFilterChain->run()
#12 D:\YiiRoot\web\CController.php(265): PostController->runActionWithFilters()
#13 D:\YiiRoot\web\CWebApplication.php(282): PostController->run()
#14 D:\YiiRoot\web\CWebApplication.php(141): CWebApplication->runController()
#15 D:\YiiRoot\base\CApplication.php(180): CWebApplication->processRequest()
#16 D:\Webroot\test.loc\index.php(11): CWebApplication->run()
REQUEST_URI=/post/admin
